### PR TITLE
Taskv3

### PR DESCRIPTION
**Pull request by jgielstra-cs**
# Checklist:
Please check you fulfill ALL of the relevant checkboxes
- [x ] Notified docs of any potential USER-facing changes
- [x] Added short description of the change - with relevant motivation and context. 
- [ ] Branch has the ticket number in its name (along with a ticket summary)
- [x] Commented the code, particularly in hard-to-understand areas
- [X] Added tests that prove my fix is effective or that my feature works
- [X] Ran `npm test` and it passes
- [X] Changes generate no new warnings
- [X] Changes are backward compatible with older clusters

### CODE REVIEW

File: bin/cortex-tasks.js [hunk 1]
```diff
@@ -102,6 +112,36 @@ program
         }
     }));
 
+// Pause a task
+program
```
Comment (clee-cs) [inline bin/cortex-tasks.js:116]: pause and resume on scheduled tasks is basically undeploy and deploy?

File: bin/cortex-agents.js [hunk 2]
```diff
@@ -138,6 +140,37 @@ program
...
+        try {
+            new DeleteAgentCommand(program).execute(agentName, options);
+        } catch (err) {
+            console.error(chalk.red(err.message));
+        }
+    }));
+
+// Undeploy agent
+program
+    .command('undeploy <agentNames...>')
```
Comment (clee-cs) [inline bin/cortex-agents.js:161]: was kinda wondering if we should add a `--all` option for our undeploy commands to make it easier to cleanup all deployed resources for a project?..

File: src/commands/tasks.js [hunk 3]
```diff
@@ -16,12 +16,33 @@
...
+const dayjs = require('dayjs');
+const relativeTime = require('dayjs/plugin/relativeTime');
 const { loadProfile } = require('../config');
 const Tasks = require('../client/tasks');
-
 const { printSuccess, printError, handleTable } = require('./utils');
 
+dayjs.extend(relativeTime);
+
+const TASK_LIST_TABLE = [
```
Comment (clee-cs) [inline src/commands/tasks.js:27]: I know we've got a few integration tests that check the table formats for us, should we toss in (or plan to) some unit tests to verify the table formatting we're expecting?.. 
Not saying we should be doing hardcoded string assertions like the int tests had scattered around, but maybe we can double check column placement, truncation, printing nested objects from valid backend responses.. just spitballing

**Review by clee-cs — APPROVED** [review]
LGTM, couple small comments for my own understanding.. trying to keep conversations out of slack DM's :)

File: bin/cortex-tasks.js (hunk 1 above)
Comment (jgielstra-cs) [reply bin/cortex-tasks.js:116]: This is pause the schedule and resume schedule ...  this needed some tweaks ...

File: bin/cortex-agents.js (hunk 2 above)
Comment (jgielstra-cs) [reply bin/cortex-agents.js:161]: Put a ticket in ... all in project isn't a bad idea..

File: src/commands/tasks.js (hunk 3 above)
Comment (jgielstra-cs) [reply src/commands/tasks.js:27]: We have unit tests             checkAll(tasksResp, checkTaskListCommonmEntry);

File: bin/cortex-agents.js [hunk 4]
```diff
@@ -138,6 +140,37 @@ program
         }
     }));
 
+// Delete Agent
```
[the remark this answers is not in the document]
Comment (jgielstra-cs) [reply bin/cortex-agents.js:143]: Just moved them around , trying to group them better...

File: bin/cortex-agents.js (hunk 2 above)
Comment (clee-cs) [reply bin/cortex-agents.js:161]: https://cognitivescale.atlassian.net/browse/FAB-4875

**Pull request merged**